.docs/user-guide/storage-providers.md

```diff
@@ -739,36 +743,45 @@ gcepd:
 
 #### Configuration Notes
 
-* The `keyfile` parameter is required. It specifies a path on disk to a file
-  containing the JSON-encoded service account credentials. This file can be
-  downloaded from the GCE web portal.
+* The `keyfile` parameter is optional. It specifies a path on disk to a file
+  containing the JSON-encoded Service Account credentials. This file can be
+  downloaded from the GCE web portal. If `keyfile` is specified, the GCE
+  instance's service account is not considered, and is not necessary. If
+  `keyfile` is *not* specified, the application will try to lookup
+  [application default credentials](https://developers.google.com/identity/protocols/application-default-credentials).
+  This has the effect of looking for credentials in the priority described
+  [here](https://godoc.org/golang.org/x/oauth2/google#FindDefaultCredentials).
 * The `zone` parameter is optional, and configures the driver to *only* allow
   access to the given zone. Creating and listing disks from other zones will be
   denied. If a zone is not specified, the zone from the client Instance ID will
   be used when creating new disks.
-* The `defaultDiskType` parameter is optional, and specified what type of disk
+* The `defaultDiskType` parameter is optional, and specifies what type of disk
```

Comment:
Hi @codenrhoden,

This should omit the comma as follows:

> The `defaultDiskType` parameter is optional and specifies what type of disk

The secondary part of the sentence is not a sentence on its own and thus a conjunctive comma is inappropriate.